<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">EE Save Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resource.qrc">
    <normaloff>:/img/shield.ico</normaloff>:/img/shield.ico</iconset>
  </property>
  <property name="toolTip">
   <string>A save editor for the Enhanced Editions of the IE games</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSettings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuDiscord">
     <property name="title">
      <string>Discord</string>
     </property>
     <property name="icon">
      <iconset resource="../../resources/resource.qrc">
       <normaloff>:/img/discord.png</normaloff>:/img/discord.png</iconset>
     </property>
     <addaction name="actionGibberlings"/>
     <addaction name="actionInfinityEngine"/>
    </widget>
    <addaction name="actionAbout"/>
    <addaction name="menuDiscord"/>
    <addaction name="actionForum"/>
    <addaction name="actionGitHub"/>
    <addaction name="actionRoyalProtector_s_Mods"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::ToolBarArea::NoToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openFromToolbar"/>
   <addaction name="saveFromToolbar"/>
   <addaction name="actionReload"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderNew"/>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/information.png</normaloff>:/img/information.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionForum">
   <property name="text">
    <string>Forum...</string>
   </property>
  </action>
  <action name="actionGibberlings">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/g3.png</normaloff>:/img/g3.png</iconset>
   </property>
   <property name="text">
    <string>Gibberlings3...</string>
   </property>
   <property name="toolTip">
    <string>Join the biggest IE community on Discord!</string>
   </property>
  </action>
  <action name="actionInfinityEngine">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/bg.png</normaloff>:/img/bg.png</iconset>
   </property>
   <property name="text">
    <string>Infinity Engine...</string>
   </property>
   <property name="toolTip">
    <string>Join the biggest IE community on Discord!</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/github.png</normaloff>:/img/github.png</iconset>
   </property>
   <property name="text">
    <string>GitHub...</string>
   </property>
   <property name="toolTip">
    <string>Discord servers where you can find me and many other enthusiasts of the games</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/out.png</normaloff>:/img/out.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRoyalProtector_s_Mods">
   <property name="text">
    <string>RoyalProtector's Mods</string>
   </property>
  </action>
  <action name="openFromToolbar">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/open-folder.png</normaloff>:/img/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a saved game</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="saveFromToolbar">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/diskette.png</normaloff>:/img/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save changes made to the game</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/diskette.png</normaloff>:/img/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/refresh_2.png</normaloff>:/img/refresh_2.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload the game's resources</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../resources/resource.qrc">
     <normaloff>:/img/setting.png</normaloff>:/img/setting.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
